frontend_scv/assets/Todo.dart

```diff
@@ -1,16 +1,13 @@
 /*
-* TODO:
 * ---------------
-* Evidence
-* Contact List
-* Payment condition must be more flexible (state which party pays)
-* Generic API Handler (Hairdu example)
-* Add payment condition to blockchain (Based on backend payment condition)
-* Touch Juror screen's Action Area
-* Pay Platform once accepted on blockchain
-* Duration input -> DatePicker
-* Jury chat --> Double Check
-* In the contacts screen, account for uppercase to lowercase walletID conversion
-* Also restyle contacts screen.
-* TODO: Go through all TODO's
+* Evidence - touch up, make Judges not able to add evidence
```

Comment:
Is this list not suppose to be empty?

Reply:
Yes, but we didn't have 'empty Todo list' on the todo list